services/app-api/forms/mcpar.json

```diff
@@ -3313,7 +3313,7 @@
             "enterEntityDetailsButtonText": "Enter measure results",
             "editEntityDetailsButtonText": "Edit measure results",
             "drawerEyebrowTitle": "D2.VII.9a and D2.VII.9b",
-            "drawerTitle": "Add plan-level measure results",
+            "drawerTitle": "plan-level measure results",
```

Comment:
do we need the `{{action}}` insert here?
![Screenshot 2025-04-02 at 10 43 03 AM](https://github.com/user-attachments/assets/80fcdfb2-30fb-4083-97ed-c21a1155e907)